jni/src/main/java/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.java

```diff
@@ -178,9 +178,8 @@ public ConfigurationBuilder marshaller(Class<? extends Marshaller> marshaller) {
 
```

Comment:
Should this^^^ version of the method work too?

Reply:
ok enabled